roles/util-cfg-service/templates/dewey.properties.j2

```diff
@@ -4,6 +4,7 @@ dewey.amqp.uri                  = {{ amqp_irods_uri }}
 dewey.amqp.exchange.name        = {{ amqp.irods.name }}
 dewey.amqp.exchange.durable     = {{ amqp.irods.durable }}
 dewey.amqp.exchange.auto-delete = {{ amqp.irods.auto_delete }}
+dewey.amqp.queue.name           = {{ dewey.amqp.queue_name }}
```

Comment:
Same comment here as on the dewey PR -- the code that reads from the config seems to expect `queue_name`, not `queue.name`, so one way or the other this should be made consistent.